cache/redis_cache.go

```diff
@@ -23,7 +24,7 @@ const getTimeout = 1 * time.Second
 const putTimeout = 2 * time.Second
 const statsTimeout = 500 * time.Millisecond
 
-type redisCachePayload struct {
+type RedisCachePayload struct {
```

Comment:
This data type should stay internal to cache package. I understand that it's convenient to expose it outside of the package for the test scenario, but it can bite us back. 

Could you make it private again and for now have a local struct in the test to assert desired fields? 🙏 

We're working on improving tests structure and code coverage which will make us move the inspection of cached data to the responsible unit tests.

Reply:
Agreed and done👌🏻